From b81d0d64564d09203b8ae0131c539d2e9d9c4ef8 Mon Sep 17 00:00:00 2001
From: Andrei Kholodnyi <andrei.kholodnyi@gmail.com>
Date: Tue, 1 Dec 2015 13:06:43 +0800
Subject: [PATCH 1/2] talloc: remove default packages from the packages list
 since empty

[ Upstream status: Changes Requested, archived ]
Patch origin: http://patchwork.openembedded.org/patch/104719/

populate_sdk fails with following message:
error: talloc-dev-2.1.1-r0 requires talloc = 2.1.1-r0

these packages are empty,remove them from the package list

Signed-off-by: Andrei Kholodnyi <andrei.kholodnyi@gmail.com>
Signed-off-by: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
---
 .../meta-networking/recipes-support/talloc/talloc_2.1.1.bb           | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/meta-openembedded/meta-networking/recipes-support/talloc/talloc_2.1.1.bb b/meta-openembedded/meta-networking/recipes-support/talloc/talloc_2.1.1.bb
index 89c390f..0c98bb6 100644
--- a/meta-openembedded/meta-networking/recipes-support/talloc/talloc_2.1.1.bb
+++ b/meta-openembedded/meta-networking/recipes-support/talloc/talloc_2.1.1.bb
@@ -20,12 +20,9 @@ EXTRA_OECONF += "--disable-rpath \
                  --with-libiconv=${STAGING_DIR_HOST}${prefix}\
                 "
 
-PACKAGES += "libtalloc libtalloc-dbg libtalloc-dev pytalloc pytalloc-dbg pytalloc-dev"
+PACKAGES = "libtalloc libtalloc-dbg libtalloc-dev pytalloc pytalloc-dbg pytalloc-dev"
 
 #ALLOW_EMPTY_${PN} = "1"
-FILES_${PN} = ""
-FILES_${PN}-dev = ""
-FILES_${PN}-dbg = ""
 
 FILES_libtalloc = "${libdir}/libtalloc.so.2 \
                    ${libdir}/libtalloc.so.2.1.1"
-- 
1.9.1

